README.md

```diff
@@ -2113,6 +2114,51 @@ If you want to sort the result before paging, you need to specify a comparator o
 
 Also, You can access a specific page more easily with the help of the `setPage` function. This way, if you make a query for the hundredth page, for example, it will get all 100 pages at once instead of reaching the hundredth page one by one using the `nextPage` function.
 
+### 7.7.2. Fast-Aggregations
+
+Fast-Aggregations are equivalent to the MapReduce Aggregators in most of the use cases, but instead of running on the MapReduce engine they run on the Query infrastructure. Their performance is tens to hundreds times better since they run in parallel for each partition and are highly optimized for speed and low memory consumption.
```

Comment:
This paragraph is out of context. We did not mention anything named MapReduce until this point. Performance and memory consumption does not have any meaning. I'd tell what aggregators do instead. Also mentioning fast-aggregations running parallel is ok.